packaging/deb/grafana-agent.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Monitoring system and forwarder
+Documentation=https://github.com/grafana/agent/blob/master/docs/README.md
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Restart=always
+User=grafana-agent
+EnvironmentFile=/etc/default/grafana-agent
+ExecStart=/usr/bin/grafana-agent --config.file $CONFIG_FILE --log.level $LOG_LEVEL $CUSTOM_ARGS
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
Could you explain how this works? The Agent doesn't actually respond to a SIGHUP right now like Prometheus does, so it would kill the process. (See #147)

Reply:
Let me do some testing for this.

Reply:
In fact, I think this should just be removed unless there is a way to trigger the agent to reload its configuration (ideally with some synchronous call)?

Reply:
No way to trigger a reload yet, it currently requires a hard restart.